ToyBox/Info.json

```diff
@@ -2,7 +2,7 @@
   "Id": "0ToyBox0",
   "DisplayName": "ToyBox",
   "Author": "Narria",
-  "Version": "1.7.5",
+  "Version": "1.8.0",
```

Comment:
1.8.0? Is that such a major feature?

Reply:
I'm just following SemVer

Reply:
I think that's too late for ToyBox.
I'm glad I decided to stop lettering, but now upping the minor version for every single feature might just confuse users, even if that's what SemVer would dictate